Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+readTrusted 'Dockerfile_jenkins_ubuntu32'
+
+pipeline {
+  agent none
+
+  stages {
+    stage('64bit') {
+      agent {
+        docker {
+          image 'golang:1.9.2'
+        }
+      }
+
+      steps {
+        sh """
+        mkdir -p /go/src/github.com/influxdata
+        cp -a $WORKSPACE /go/src/github.com/influxdata/influxdb
+
+        cd /go/src/github.com/influxdata/influxdb
+        go get github.com/sparrc/gdm
+        gdm restore
+        """
+
+        sh """
+        cd /go/src/github.com/influxdata/influxdb
+        go test -parallel=1 ./...
```

Comment:
Why are we running with `parallel=1`? That's only going to run one package's tests at a time.

Reply:
We are currently doing that for CI on Circle. It's supposed to make sure the output is properly aligned, but I just recently learned that it doesn't do that so this will likely change.

Reply:
This is how many concurrent `t.Parallel` tests are run.

```
	-parallel n
	    Allow parallel execution of test functions that call t.Parallel.
	    The value of this flag is the maximum number of tests to run
	    simultaneously; by default, it is set to the value of GOMAXPROCS.
	    Note that -parallel only applies within a single test binary.
	    The 'go test' command may run tests for different packages
	    in parallel as well, according to the setting of the -p flag
	    (see 'go help build').
```

If we needed one package to run at a time, the `-p` help says:

```
	-p n
		the number of programs, such as build commands or
		test binaries, that can be run in parallel.
		The default is the number of CPUs available.
```

Reply:
@mark-rushakoff I'm always getting `-parallel` and `-p` mixed up 😄